src/nano_contracts/utils.ts

```diff
@@ -144,8 +151,18 @@ export const validateBlueprintMethodArgs = async (blueprintId, method, args): Pr
     throw new NanoContractTransactionError(`Blueprint does not have method ${method}.`);
   }
 
-  // Args may come as undefined
-  const argsLen = args ? args.length : 0;
+  // Args may come as undefined or null
+  if (args == null) {
+    if (methodArgs.length !== 0) {
+      throw new NanoContractTransactionError(
```

Comment:
chore: We should add a test to validate this protection

Reply:
Done https://github.com/HathorNetwork/hathor-wallet-lib/pull/729/commits/3009dce76ecd371c1e3d1630a58e856b164afb80